.github/workflows/test_suite_ubuntu.yml

```
@@ -99,7 +99,12 @@ jobs:
           cmake --build .
           cmake --install .
 
-      - name: Test suite
+      - name: Run unit tests
         run: |
           . ftorch/bin/activate
-          ./run_test_suite.sh -V
+          ./run_test_suite.sh -V -u
+
+      - name: Run integration tests
+        run: |
+          . ftorch/bin/activate
+          ./run_test_suite.sh -V -i
```

Some advice I had from @christopheredsall always use the verbose flags in bash scripts or CI jobs as it is easy in future to see what it is doing, rather than looking up further docs, and reduces risk of wrong flags creeping in.

```suggestion
          ./run_test_suite.sh --verbose --unit-only

      - name: Run integration tests
        run: |
          . ftorch/bin/activate
          ./run_test_suite.sh --verbose --integration-only
```